lib/challenge.js

```diff
@@ -48,7 +57,13 @@ var pet_owner;
   // Every pet can:
   // - make noise (each pet makes its own unique noise, as specified by `noise`.
 
-var some_pet;
+var some_pet = {
+  name:'Baxter',
+  species: 'dog',
+  breed:'Golden Retriver'
+  noise: 'bark',
+  makeNoise:
```

Comment:
The value corresponding to the key `makeNoise` is expected to be a function. Properties/attributes of objects that point to functions are called methods.